pkg/cmd/properties/properties.go

```diff
@@ -36,6 +41,7 @@ type Properties interface {
 	AddStringPersistentFlag(name string, defaultVal string, description string)
 	AddStringFlag(name string, description string)
 	AddDurationProperty(name string, defaultVal time.Duration, description string)
+	AddDurationRangeProperty(name string, defaultVal time.Duration, description string, lowerLimit, upperLimit time.Duration)
```

Comment:
Rather than a new func here can we update the old one to have an optional set of functional arguments?

AddDurationProperty(name string, defaultVal time.Duration, description string, opts func()...)

Reply:
updated to use optional funcs... thank you for the teach